packages/lib-classifier/src/plugins/drawingTools/experimental/models/tools/FreehandLineTool/FreehandLineTool.js

```diff
@@ -8,16 +8,19 @@ const FreehandLineTool = types
     type: types.literal('freehandLine')
   })
   .actions((self) => ({
-    createMark(mark) {
+    createMark(mark, points) {
       const newMark = FreehandLine.create(
-        Object.assign({}, mark, { toolType: self.type })
+        { ...mark, toolType: self.type}
```

Comment:
```suggestion
        {
          ...mark,
          pathX: points.map(point => point.x),
          pathY: points.map(point => point.y),
          toolType: self.type
        }
```

You can pass initial values for `mark.pathX` and `mark.pathY` as part of the initial snapshot here.

Alternatively, add them to the mark before calling `freehandTool.createMark(mark)`, which would save you from having to pass a separate `points` arg and keep the API for this tool the same as all the other drawing tools. You can then read the points from `self.pathX` and `self.pathY` when you initialise volatile state.

Reply:
Got it - made the change so it can handle either scenario and keeps the same API interface.